lua-http-parser.c

```diff
@@ -213,7 +249,12 @@ static int lhp_push_nil_event(lhttp_parser* lparser, int cb_id) {
     if ( ! lua_checkstack(L, 5) ) return -1;
 
     lua_rawgeti(L, ST_FENV_IDX, cb_id);
-    lua_pushnil(L);
+    if(CB_ON_CHUNK_HEADER == cb_id){
+      lhp_pushint64(L, lparser->parser.content_length);
```

Comment:
May be it worth stop with error if we can not represent `content_length` as Lua number.
Also may be add separate method to read this property?

Reply:
Don't throw an error.  Even Lua 5.1 doesn't support returning 64bit lengths (#string, #table, etc...) return a Lua Number.  The Lua 5.1 core VM will truncate the length if it is larger then what a Lua Number can hold, no error.

If someone really wants to handle content that large, they can use a newer Lua version.